main.go

```diff
@@ -45,7 +45,16 @@ func validate(n string) {
 
 func display_commit_hash_and_message() {
 	/* Display commit hash and message. The [pickup|..] strings is colored */
-	for i := len(commitMsg) - 2; i >= 0; i-- {
+
+	/* set limit to display the history */
+	var limit int
+	if iNum+2 > len(commitMsg)-2 {
```

Comment:
go fmt removed space 🤔

Reply:
Yes. I think we should conform to the golang tool proposal.